web/src/components/troubleshoot/SupportBundleRow.tsx

```diff
@@ -258,7 +274,7 @@ class SupportBundleRow extends React.Component {
                 onClick={() => this.handleBundleClick(bundle)}
               >
                 <div className="flex">
-                  {!this.props.isCustomer && bundle.customer ? (
```

Comment:
as far as i can [tell](https://github.com/replicatedhq/kots/blob/939172d803075c2b79ad5d966f573c8e6df44175/pkg/supportbundle/types/types.go#L16-L36) there is no `customer` on the SupportBundle response. This comes straight off of the response and i didn't see anywhere where a customer was added to the response data by the FE code

Reply:
good catch